src/mlmpfr.ml

```diff
@@ -760,10 +758,10 @@ let get_formatted_str ?(rnd = To_Nearest) ?(base = 10) ?(size = 0) ?(ktz = true)
     Printf.sprintf "%s%s%s%c%+03d"
       (if neg then String.sub mantissa 0 2 else Char.escaped mantissa.[0])
       (if
-       (neg && String.length mantissa == 2)
-       || (neg == false && String.length mantissa == 1)
-      then ""
-      else ".")
+         (neg && String.length mantissa == 2)
```

Comment:
Please stick to the original coding style. Use ocp-indent for OCaml files (you can also run clang-format on C sources).